service/src/main/java/bio/terra/pipelines/stairway/imputation/steps/gcp/SubmitCromwellSubmissionStep.java

```diff
@@ -59,10 +71,65 @@ public StepResult doStep(FlightContext flightContext) {
     String description = inputParameters.get(JobMapKeys.DESCRIPTION.getKeyName(), String.class);
     String wdlMethodName =
         inputParameters.get(RunImputationJobFlightMapKeys.WDL_METHOD_NAME, String.class);
+    String wdlMethodVersion =
+        inputParameters.get(RunImputationJobFlightMapKeys.WDL_METHOD_VERSION, String.class);
+
+    List<PipelineInputDefinition> inputDefinitions =
+        inputParameters.get(
+            RunImputationJobFlightMapKeys.PIPELINE_INPUT_DEFINITIONS, new TypeReference<>() {});
+    List<PipelineOutputDefinition> outputDefinitions =
+        inputParameters.get(
+            RunImputationJobFlightMapKeys.PIPELINE_OUTPUT_DEFINITIONS, new TypeReference<>() {});
 
     // validate and extract parameters from working map
     FlightMap workingMap = flightContext.getWorkingMap();
 
+    // grab current method config and validate it
+    MethodConfiguration methodConfiguration =
+        rawlsService.getCurrentMethodConfigForMethod(
+            samService.getTeaspoonsServiceAccountToken(),
+            controlWorkspaceProject,
+            controlWorkspaceName,
+            wdlMethodName);
+    boolean validMethodConfig =
+        rawlsService.validateMethodConfig(
+            methodConfiguration,
+            wdlMethodName,
+            inputDefinitions,
+            outputDefinitions,
+            wdlMethodVersion);
+
+    // if not a valid method config, set the method config to what we think it should be.  This
+    // shouldn't happen
```

Comment:
shouldn't happen often, but it will happen once every time we change the wdlMethodVersion... unless we update it in Terra as well as in our service, which actually we could do. 🤷 it's nice to have the warning log here. wonder if we should set up a slack channel to capture warnings.

Reply:
is that something we can do -- is that a sentry thingy?

Reply:
yeah probably sentry